server/api.proto

```diff
@@ -156,6 +156,11 @@ message Envelope {
     TLeaderboards leaderboards = 57;
     TLeaderboardRecord leaderboard_record = 58;
     TLeaderboardRecords leaderboard_records = 59;
+
+    TMatchmakingStart matchmaking_start = 60;
```

Comment:
I prefer `TMatchmaking*` over `TMatchmake`, but I can get behind `Add` and `Remove` instead of `Start` and `Cancel` if you and @mofirouz feel it works better. `TMatchmakingMatched` (or `TMatchmakeMatched`) sounds a little awkward to me. @mofirouz wdyt?